assets/js/sections/bandmap_list.js

```diff
@@ -24,7 +24,7 @@ $(function() {
 					'targets': 2,
 					'createdCell':  function (td, cellData, rowData, row, col) {
 						$(td).addClass("spotted_call"); 
-						// $(td).attr( "title", "Click to prepare logging" );
+						$(td).attr( "title", "Click to prepare logging" );
 					}
```

Comment:
Let's make this translatable. Code in `application/views/bandmap/index.php` at the top where already are some javascript variables

```javascript
var lang_click_to_prepare_logging = "<?= __("Click to prepare logging."); ?>";
```

and set here 


```suggestion
						$(td).attr( "title", lang_click_to_prepare_logging );
```